src/containers/questionCard/questionCard.js

```diff
@@ -1,27 +1,61 @@
 import React, { useEffect, useState } from "react";
 import Question from "../../components/question";
-import FetchQuestions from "../../components/question/DataFetching";
+import axios from "axios";
+import ResultCard from "../../components/result/resultCard"
 
-export const QuestionContainer = () => {
+let api_url = "https://the-trivia-api.com/api/questions";
+function QuestionContainer(){
+    let [question,setQuestions]= useState([])
+    let [index,setIndex] = useState({});
+    let [score,setScore] = useState(0);
+    useEffect(()=>{
+        if(question.length === 0){
+            try{
+                axios.get(api_url)
+            .then((res) => {
+                setQuestions(res.data);
+                setIndex(0);
+            })
+            .catch((err) => {
+                return err;
+            });
+            }catch(err){console.log("Error 404")}
+        }
+    },[question]);
+    let handleClick = () =>{
+        return question[index].correctAnswer;
+    }
+    function incrementIndex(){
+        if(index < 10){
+            console.log(question[index].correctAnswer)
+            if(question[index].correctAnswer === handleClick()){
```

Comment:
you can save the handleClick() result in a variable and compare it to the correctAnswer